targets/trusted_root.json

```diff
@@ -0,0 +1,92 @@
+{
+  "mediaType": "application/vnd.dev.sigstore.trustedroot+json;version=0.1",
+  "tlogs": [
+    {
+      "baseUrl": "https://rekor.sigstore.dev",
+      "hashAlgorithm": "SHA2_256",
+      "publicKey": {
+        "rawBytes": "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE2G2Y+2tabdTV5BcGiBIx0a9fAFwrkBbmLSGtks4L3qX6yYY0zufBnhC8Ur/iy55GhWP/9A/bY2LhC30M9+RYtw==",
+        "keyDetails": "PKIX_ECDSA_P256_SHA_256",
+        "validFor": {
+          "start": "2020-06-17T12:44:14.000Z"
```

Comment:
Need better start date here (the value here is the timestamp of the first commit in the Rekor repo)

Reply:
Interesting, the first item on the log was on `2021-01-12T11:53:27Z`.... maybe that? there shouldn't be a reason to verify anything before then. @bobcallaway is this right?

Reply:
sounds right

Reply:
Updated to match the timestamp of the first item in the log.